okteto-pipeline.yml

```diff
@@ -1,5 +0,0 @@
-icon: https://github.com/okteto/go-getting-started/raw/master/icon.png
```

Comment:
why are we removing this?

Reply:
To follow the idea of having samples for single features. This sample is about `okteto init` and `okteto up`. We use it for tutorials on mikube too.